src/test/java/edu/oakland/test/database/DatabaseCommInterfaceTest.java

```diff
@@ -0,0 +1,119 @@
+package edu.oakland.test.database;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import edu.oakland.helper.admin.LocationDataPoint;
+import edu.oakland.helper.admin.TrackData;
+import edu.oakland.production.database.DatabaseCommInterface;
+import edu.oakland.production.database.DatabaseCommInterfaceClass;
+import edu.oakland.production.database.DatabasePersistentStorage;
+import java.lang.IllegalArgumentException;
+import java.time.LocalDateTime;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("Database CommInterface Test")
+public class DatabaseCommInterfaceTest {
+
+  @Test
+  @DisplayName("rfid going in is Rfid coming Out")
+  void rfidInIsRfidOut() {
+    int i = 10;
```

Comment:
Let's make this random to increase test robustness:
```suggestion
    int i = (int) (Math.random() * 100);
```